packages/@react-spectrum/table/test/Table.test.js

```diff
@@ -4259,9 +4259,9 @@ describe('TableView', function () {
 
         for (let row of rows) {
           expect(row.childNodes[0].style.width).toBe('38px');
-          expect(row.childNodes[1].style.width).toBe('320.6666666666667px');
-          expect(row.childNodes[2].style.width).toBe('320.6666666666667px');
-          expect(row.childNodes[3].style.width).toBe('320.6666666666667px');
+          expect(row.childNodes[1].style.width).toBe('320.67px');
+          expect(row.childNodes[2].style.width).toBe('320.66px');
+          expect(row.childNodes[3].style.width).toBe('320.67px');
```

Comment:
Currently rounding to 2 decimal places for predictability. We're seeing a visual bug that might be fixed if we just round to whole numbers though, so we are considering doing that if there's no reason to be rounding to 2 decimal places and if it actually does fix the bug.

Reply:
Okay we may just round then. I'll test it out when I have a chance. We were rounding initially and then realized that tests failed because they were expecting explicit non-rounded pixel results so we wondered if there was a reason that rounding wasn't happening. I think as far as pixels go though, rounding makes the most sense.